cmd/hypper/upgrade.go

```diff
@@ -145,7 +146,7 @@ func newUpgradeCmd(cfg *action.Configuration, logger log.Logger) *cobra.Command
 					instClient.Description = client.Description
 					instClient.ReleaseName = client.ReleaseName
 
-					rel, err := runInstall(args, instClient, valueOpts, logger)
+					rel, err := runInstall(args, instClient, valueOpts, logger, settings)
```

Comment:
why pass settings here? it is a global var in the package main.

Reply:
Good catch!
fixed in https://github.com/rancher-sandbox/hypper/pull/120/commits/dc715749485579715a1662cc9e752820307da933.